code/game/machinery/Sleeper.dm

```diff
@@ -33,6 +33,11 @@
 
 /obj/machinery/sleeper/examine(mob/user)
 	. = ..()
+	if(occupant)
+		if(occupant.is_dead())
+			. += "<span class='warning'>You see [occupant.name] inside. [occupant.p_they(TRUE, FALSE)] is dead!</span>"
```

Comment:
I did some diving, and `occupant.name` itself is indeed updated on human life, so we thankfully don't need to check if they're hidden or not.
Only other thing is that I'm not sure why you're passing `FALSE` in for the temp gender (second argument). For humans, it gets overridden by `get_visible_gender()` anyway, so I'm pretty sure it's unnecessary.

Anyway, please make these sorts of changes through all the instances in the PR.
```suggestion
			. += "<span class='warning'>You see [occupant.name] inside. [occupant.p_they(TRUE)] [occupant.p_are()] dead!</span>"
```